File: python/1010.py
# -*- coding: utf-8 -*-

p1_code, p1_number, p1_price = input().split()

p1_code = int(p1_code)
p1_number = int(p1_number)
p1_price = float(p1_price)

p2_code, p2_number, p2_price = input().split()

p2_code = int(p2_code)
p2_number = int(p2_number)
p2_price = float(p2_price)

result = (p1_number * p1_price + p2_number * p2_price)

print("VALOR A PAGAR: R$ %.2f" %result)

File: src/javascript/1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var first = lines.shift().split(' ');
var second = lines.shift().split(' ');

var n1 = parseInt(first[1]);
var v1 = parseFloat(first[2]);

var n2 = parseInt(second[1]);
var v2 = parseFloat(second[2]);

var total = n1 * v1 + n2 * v2;
console.log('VALOR A PAGAR: R$ ' + (total).toFixed(2));